### ISSUE TITLE
With Respect to Fractal Properties...

### ISSUE THREAD

**Issue by tamos** [uchicago-computation-workshop/marc_berman#3]
Specifically, I am thinking of two points: 

i) Ubiquity:  Almost all natural phenomena display at least some fractal properties - thus the fractal nature of a given phenomenon in the brain may in fact be shared across many phenomena in the brain/body. 

ii) Flexibility: Fractal analyses of natural phenomena often afford substantial latitude when it comes to fractal properties. For example, the canonical case of the UK coastline is not strictly a fractal, unless you adopt a relaxed idea of self-similiarity. 

What are the implications of both/either of these items for your work?

**Comment by liu431** [comment]
I agree with you that almost all natural phenomena display at least some fractal properties. Mathematicians have studied the properties of fractal for a long time, and it's exciting to explore the fractal in the brain.

**Comment by bermanm** [comment]
Right, most natural phenomena are not perfect statistical fractals, but that does not mean that there still is not large variability.  We are looking at some of this now with architecture and how different types of architecture have greater fractalness, and are judged to be more preferred, natural and comforting.  We would then wonder whether perceiving that fractalness might put the brain into more fractal states

**Comment by Fangfangwan** [comment]
Good point!
